boost-maven/boost-maven-plugin/src/it/test-mpJwt-1.1/src/main/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,13 @@
+<!-- ?xml version="1.0" encoding="UTF-8"?>
+<  beans xmlns="http://java.sun.com/xml/ns/javaee"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="
+        http://java.sun.com/xml/ns/javaee
+        http://java.sun.com/xml/ns/javaee/beans_1_2.xsd"
+        version="1.2" bean-discovery-mode="all">
+       <scan>
+        <exclude name="org.eclipse.microprofile.jwt.JsonWebToken">
+        </exclude>
+       </scan>
+</beans -->
+<beans/>
```

Comment:
Hmm...I wouldn't expect you'd need an empty beans.xml at this CDI level, but I'd expect to see a QName not just an unqualified `<beans>` element.  Maybe we don't need this?

Reply:
Thought I needed this for CDI to work for tomee.. Figrured out I didn't and forgot to remove it. Will delete.